mobile/ios/WirelessDebug/WirelessDebugger/WebSocketMessenger.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  WebSocketMessenger.swift
+//  Wireless Debug iOS Test App
+//
+
+import UIKit
+import Foundation
+
+/// Handles opening a WebSocket connection and sending the necessary messages
+/// across that connection.
+class WebSocketMessenger {
```

Comment:
The idea here is that the server could change. For example, someone could implement their own custom server which accepts WebSocket connections and then use this library to point to it. I chose this name because it generalizes the underlying purpose of this class which is to send WebSocket messages.